static/app/components/replays/table/replayTableColumns.tsx

```diff
@@ -522,18 +523,40 @@ export const ReplaySessionColumn: ReplayTableColumn = {
 
     const referrer = getRouteStringFromRoutes(routes);
     const eventView = EventView.fromLocation(location);
+
+    const {statsPeriod, ...eventViewQuery} = eventView.generateQueryStringObject();
+
+    let query = {
+      referrer,
+      ...eventViewQuery,
+    };
+
+    if (!eventViewQuery.start && !eventViewQuery.end && typeof statsPeriod === 'string') {
+      const {start: playlistStart, end: playlistEnd} = parseStatsPeriod(statsPeriod);
+      query = {...query, ...{playlistStart, playlistEnd}};
```

Comment:
m: no need to spread `query` - you can assign directly on the object

```suggestion
      query.playlistStart = playlistStart;
      query.playlistEnd = playlistEnd;
```